castle/cms/utils.py

```diff
@@ -485,9 +485,9 @@ def get_ip(req):
 def get_backend_url():
     registry = getUtility(IRegistry)
     backend_url = registry.get('plone.backend_url', None)
-    if not backend_url:
+    if not backend_url or len(backend_url) == 0:
         backend_url = api.portal.get().absolute_url()
-    return backend_url
+    return backend_url[0]
```

Comment:
if line 489 happens, I think this will return the first character of the portal URL